### PR TITLE
Vendor GIDD

### PR DESCRIPTION
**Pull request by Geevies**
* Vendors have given some glow in the dark capabilities.
* Vendors now output to your hands, or to your feet, if your hands are full.

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -583,7 +604,14 @@
 
 	if((!allowed(usr)) && !emagged && scan_id)	//For SECURE VENDING MACHINES YEAH
 		to_chat(usr, "<span class='warning'>Access denied.</span>")	//Unless emagged of course)
-		flick(src.icon_deny,src)
+		if(icon_deny_cuts_overlay)
+			cut_overlays(TRUE)
+			cut_overlays(TRUE)
+			addtimer(CALLBACK(src, .proc/add_screen_overlay), deny_time ? deny_time : 15)
+			cut_overlays(TRUE)
```
Comment (alsoandanswer) [inline code/game/machinery/vending.dm:611]: is it supposed to cut_overlays 3 times?

Comment (Geevies) [reply code/game/machinery/vending.dm:611]: yes and no

Comment (alsoandanswer) [reply code/game/machinery/vending.dm:611]: what

**Pull request merged**